retrievalmarket/impl/client.go

```diff
@@ -188,7 +163,7 @@ type clientStream struct {
 	verifier   BlockVerifier
 	bs         blockstore.Blockstore
 }
-
+*/
 /* This is the old retrieval code that is NOT spec compliant */
```

Comment:
same re: commented out code

Reply:
do not delete in this case -- it's the whole old implementation -- need it for reference